<launch>
    <node pkg="my_turtlebot3_vision" type="face_tracker.py" name="face_tracker" output="screen">

        <remap from="input_rgb_image" to="/image_raw" />
        <remap from="input_depth_image" to="/camera/depth/image_raw" />
    
        <rosparam>
            show_text: True
            gf_maxCorners: 200
            gf_qualityLevel: 0.02
            gf_minDistance: 7
            gf_blockSize: 10
            gf_useHarrisDetector: False
            gf_k: 0.04
            haar_scaleFactor: 1.3
            haar_minNeighbors: 3
            haar_minSize: 30
            haar_maxSize: 150
            feature_size: 1
            show_text: True
        </rosparam>
    
        <param name="cascade_1" value="$(find my_turtlebot3_vision)/data/haar_detectors/haarcascade_frontalface_alt2.xml" />
      	<param name="cascade_2" value="$(find my_turtlebot3_vision)/data/haar_detectors/haarcascade_frontalface_alt.xml" />
      	<param name="cascade_3" value="$(find my_turtlebot3_vision)/data/haar_detectors/haarcascade_profileface.xml" />

    </node>
</launch>